pkg/plugin/v3/scaffolds/internal/templates/config/controller/controller.go

```diff
@@ -76,6 +76,13 @@ type {{ .Resource.Kind }}Reconciler struct {
 // +kubebuilder:rbac:groups={{ .Resource.Domain }},resources={{ .Resource.Plural }},verbs=get;list;watch;create;update;patch;delete
 // +kubebuilder:rbac:groups={{ .Resource.Domain }},resources={{ .Resource.Plural }}/status,verbs=get;update;patch
 
+// Reconcile reads that state of the cluster for a {{ .Resource.Kind }} object 
+// and makes changes based on the state and what is in the {{ .Resource.Kind }}.Spec.
+// TODO(user): Modify this Reconcile function to implement your Controller logic.
+// Note:
+// The Controller will requeue the Request to be processed again if the returned
+// error is non-nil or Result.Requeue is true, otherwise upon completion it will
+// remove the work from the queue.
```

Comment:
Note that we have more requeue options. So, I am afraid that we could not add this text. 

The following are a few possible return options to restart the Reconcile:

- With the error:
return ctrl.Result{}, err

- Without an error:
return ctrl.Result{Requeue: true}, nil

- Therefore, to stop the Reconcile, use:
return ctrl.Result{}, nil

- Reconcile again after X time:
return ctrl.Result{RequeueAfter: nextRun.Sub(r.Now())}, nil

Note: For more details, check the Reconcile and its [Result godoc](https://pkg.go.dev/sigs.k8s.io/controller-runtime/pkg/reconcile).

Reply:
Added documentation pointing to the controller runtime.